manifests/service.pp

```diff
@@ -4,31 +4,49 @@
 #
 # @api private
 class puppet_metrics_dashboard::service {
-  service { $puppet_metrics_dashboard::influx_db_service_name:
-    ensure  => running,
-    enable  => true,
-    require => Package['influxdb'],
-  }
-
-  if $puppet_metrics_dashboard::enable_chronograf {
-    service { 'chronograf':
-      ensure  => running,
-      enable  => true,
-      require => [
-        Package['chronograf'],
-        Service[$puppet_metrics_dashboard::influx_db_service_name]
-      ],
+  case $facts['os']['family'] {
+    'Suse': {
+      file { '/etc/systemd/system/influxdb.service':
+        ensure => file,
+        source => '/opt/influxdb/usr/lib/influxdb/scripts/influxdb.service',
+        notify => Exec['/usr/bin/systemctl daemon-reload'],
+      }
+      exec { '/usr/bin/systemctl daemon-reload':
+        refreshonly => true,
+      }
+      service { $puppet_metrics_dashboard::influx_db_service_name:
+        ensure => running,
+        enable => true,
+      }
```

Comment:
I would think we should defer management of this to [camptocamp/systemd](https://forge.puppet.com/camptocamp/systemd)